config/database.yml

```diff
@@ -2,6 +2,8 @@ default: &default
   adapter: postgresql
   pool: 5
   timeout: 5000
+  username: <%= ENV['DATABASE_USER'] %>
+  host: <%= ENV['DATABASE_HOST'] %>
```

Comment:
That sounds like a good idea, but I think it's separate from this pull req. This will just apply the env vars regardless of how they're supplied.

The last line of the Dockerfile will copy over all resources, so if there's a .env present, it will pick it up.